dbt/adapters/trino/connections.py

```diff
@@ -286,14 +303,17 @@ def fetchone(self):
         return None
 
     def execute(self, sql, bindings=None):
-
-        if bindings is not None:
-            # trino doesn't actually pass bindings along so we have to do the
-            # escaping and formatting ourselves
+        if not self._prepared_statements_enabled and bindings is not None:
+            # DEPRECATED: by default prepared statements are used.
```

Comment:
please do create a follow-up request to remove this safety feature after 1,2 releases.